### DISCUSSION TITLE
各位开拓者你们攒了多少星琼了

### DISCUSSION THREAD

**Discussion by yangsaho** [share121/inter-knot#2090] [其他]
![1723627341236](https://github.com/user-attachments/assets/bd852470-75f8-4cb4-9448-ff0eb2b2d256)
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by icenow188** [comment]
快100抽了

**Comment by YuTinMIn** [comment]
1抽没有<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >（骄傲）

**Comment by jidianzhuxing** [comment]
57抽，不多，我打算囤到明年崩铁和fate联动的时候，消息放出来的时候我就开始攒了，等联动角色。<img src="https://img-static.mihoyo.com/communityweb/upload/eb1f17ea60f29d814dd158a486e588cb.png" alt="帕姆_叉腰" width="75" >

**Comment by ghfvhgchhfvhhtfifvb** [comment]
打算攒到和fate联动，看看联动会不会出个角色
